servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/ClientStrategyInfluencerChainBuilder.java

```diff
@@ -46,32 +59,68 @@ private ClientStrategyInfluencerChainBuilder(ClientStrategyInfluencerChainBuilde
     }
 
     void add(StreamingHttpClientFilterFactory clientFilter) {
-        clientChain = clientChain.merge(clientFilter.requiredOffloads());
+        add("filter", clientFilter, clientFilter.requiredOffloads());
     }
 
     void add(HttpLoadBalancerFactory<?> lb) {
-        clientChain = clientChain.merge(lb.requiredOffloads());
+        add("load balancer", lb, lb.requiredOffloads());
+    }
+
+    private void add(String purpose, ExecutionStrategyInfluencer<?> influencer, HttpExecutionStrategy strategy) {
+        if (defaultStrategy() == strategy || noOffloadsStrategy() == strategy) {
```

Comment:
> if we substitute `offloadAll()` we lose the ability of API conversions to substitute less offloading when default is presented.
 
Agreed, replacing it with `offloadAll()` will have other side-effects. WDYT if we keep it as-is? Merging `defaultStrategy()` with `defaultStrategy()` will result in `defaultStrategy()`, heloing us to preserve all the flags set without side-effects for merging with API conversions?

Reply:
I have removed `defaultStrategy()` as an error case for now.